src/GeneralRules/ReAssignmentOnBlockArgumentRule.class.st

```diff
@@ -32,7 +32,7 @@ ReAssignmentOnBlockArgumentRule class >> uniqueIdentifierName [
 ReAssignmentOnBlockArgumentRule >> check: aMethod forCritiquesDo: aCriticBlock [
 	| problemAssignments|
 	problemAssignments := aMethod ast assignmentNodes 
-		select: [ :assignment | assignment variable isArg].
+		select: [ :assignment | assignment variable isArgument ].
```

Comment:
Does this change anything?

Reply:
Changes that if an ASTs is installed which has no binding, this wont throw an error.
I'm not sure if ASTs can be installed by someone else but the compiler, but I think we should still go with the right semantic method when we can..